FlashCards.AhmadJer99/FlashCards/Data/DataBaseController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System.Configuration;
+using Microsoft.Data.SqlClient;
+
+namespace FlashCards.Data;
+
+internal abstract class DataBaseController<T>
```

Comment:
🟠 **Naming Conventions**

💡 As mentioned, database is one word, so use PascalCase correctly.

🔧 Rename to `DatabaseController`

Reply:
Done fixed that!